### PR TITLE
76 services descriptions

### LINKED ISSUE #76: Services descriptions
## Describe the feature
description of services in json files of the same name.

## TO DO
_Describe what it must be do for the feature_

## Extra links (Optional)
_Add any other extra links_

## Screenshots (Optional)
_Add any screenshot_

### PR DESCRIPTION
**Pull request by STom6**
# New feature
implementation of the documentation

### CODE REVIEW

File: Server/Services/description/IssStationService.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "IssStationService",
+    "actions": [
+        {
+            "name": "checkISSPosition",
```
Comment (CedricCORGE) [inline Server/Services/description/IssStationService.json]: The name shouldn't be check but compare or something like that

File: Server/Services/description/twitterService.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        {
+            "name": "Like Tweet",
+            "description": "Like the tweet of a hashtag given in parameter"
+        },
+        {
+            "name": "Retweet Tweet",
+            "description": "Retweet the tweet of a hashtag given in parameter"
+        },
+        {
+            "name": "Tweet un tweet",
```
Comment (CedricCORGE) [inline Server/Services/description/twitterService.json]: Should be Post a tweet

**Comment by osvegn** [comment]
Move this pull request as draft pull request if it isn't finished

File: Server/Services/description/IssStationService.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "IssStationService",
+    "actions": [
+        {
+            "name": "CompareIssPositionWithUserPosition",
+            "description": "checks the position of the iss in relation to the user-defined position. Compares whether the ISS is within a (user-defined) **gap** around the user. And returns whether the ISS is close to the position or not."
```
Comment (osvegn) [inline Server/Services/description/IssStationService.json]: Capital at beginning

File: Server/Services/description/OpenMeteoService.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "OpenMeteoService",
+    "actions": [
+        {
+            "name": "WeatherisFineOrNot",
+            "description": "Receive weather data for a location defined by latitude/longitude. Returns with a trigger if the weather is good or bad."
```
Comment (osvegn) [inline Server/Services/description/OpenMeteoService.json]: Receives

File: Server/Services/description/twitterService.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "actions": [
+        {
+            "name": "none",
+            "description": "none"
+        }
+    ],
+    "reactions": [
+        {
+            "name": "Like Tweet",
+            "description": "Like the tweet of a hashtag given in parameter"
```
Comment (osvegn) [inline Server/Services/description/twitterService.json]: Likes

File: Server/Services/description/twitterService.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        },
+        {
+            "name": "Retweet Tweet",
+            "description": "Retweet the tweet of a hashtag given in parameter"
```
Comment (osvegn) [inline Server/Services/description/twitterService.json]: Retweets

**Review by osvegn — CHANGES_REQUESTED** [review]
Grammar

**Review by JohanCDev — DISMISSED** [review]
Lgtm

**Review by CedricCORGE — DISMISSED** [review]

File: Server/Services/description/twitterService.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        {
+            "name": "Likes Tweet",
+            "description": "Like the tweet of a hashtag given in parameter"
+        },
+        {
+            "name": "Retweets Tweet",
+            "description": "Retweet the tweet of a hashtag given in parameter"
+        },
+        {
+            "name": "Post a tweet",
```
Comment (osvegn) [inline Server/Services/description/twitterService.json]: Posts

File: Server/Services/description/twitterService.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+            "description": "Post a tweet with a text given in parameter"
```
Comment (osvegn) [inline Server/Services/description/twitterService.json]: Posts

File: Server/Services/description/googleService.json [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+    "name": "GoogleService",
+    "actions": [
+        {
+            "name": "none",
+            "description": "none"
+        }
+    ],
+    "reactions": [
+        {
+            "name": "Send an email",
```
Comment (osvegn) [inline Server/Services/description/googleService.json]: Sends

File: Server/Services/description/IssStationService.json [hunk 10]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "IssStationService",
+    "actions": [
+        {
+            "name": "Compare Iss Position With User Position",
```
Comment (osvegn) [inline Server/Services/description/IssStationService.json]: Compares

**Review by osvegn — CHANGES_REQUESTED** [review]

**Review by 31Nathan — DISMISSED** [review]
LGTM

**Review by osvegn — DISMISSED** [review]

**Review by CedricCORGE — DISMISSED** [review]
LGTM

**Review by 31Nathan — APPROVED** [review]

**Review by osvegn — APPROVED** [review]

**Pull request merged**